resolver/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ockam-network/did"
+	"github.com/ockam-network/ockam"
+	"github.com/ockam-network/ockam/node"
+	ockamHttp "github.com/ockam-network/ockam/node/remote/http"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func run() error {
+	mux := http.NewServeMux()
+
+	port := os.Getenv("PORT")
+	discoverer_name := os.Getenv("DISCOVERER_NAME")
+	discoverer_port, err := strconv.Atoi(os.Getenv("DISCOVERER_PORT"))
+
+	if err != nil {
+		exitOnError(err)
```

Comment:
would be better to return error instead, since that is what this function expects `func run() error`